ipatests/test_integration/test_acme.py

```diff
@@ -36,6 +35,16 @@
 CERTBOT_DNS_IPA_SCRIPT = '/usr/libexec/ipa/acme/certbot-dns-ipa'
 
 
+def move_date(host, chrony_cmd, date_str):
+    """Helper method to move system date
+    :param host: host on which date is to be manipulated
+    :param chrony_cmd: state to which chrony service is to be moved
```

Comment:
:param chrony_cmd: systemctl command to apply to chrony service, for instance 'start', 'stop'